sources/packages/web/src/services/http/ApplicationApi.ts

```diff
@@ -165,4 +170,46 @@ export class ApplicationApi extends HttpBaseClient {
       this.addClientRoot(`application/${applicationNumber}/appeal`),
     );
   }
+
+  /**
+   * Get in progress details of an application by application id.
+   * @param applicationId application id.
+   * @returns application details.
+   */
+  async getInProgressApplicationDetails(
+    applicationId: number,
+  ): Promise<InProgressApplicationDetailsAPIOutDTO> {
+    const response = await this.getCall(
```

Comment:
Why are we assigning it to response? generally for get call we just return this.getCall() right?

Also we can use getcallTyped.

Same for other get calls